test.py

```diff
@@ -9,7 +9,7 @@
 from src.tokens.token_associate_transaction import TokenAssociateTransaction
 from src.transaction.transfer_transaction import TransferTransaction
```

Comment:
Here you'll have to add an admin key to the TokenCreateTransaction as well as TokenDeleteTransaction function to get this test to work - for this, you will also have to add an admin key and a set_admin_key function to the TokenCreateTransaction class. Else the delete operation won't be successful.